ide/atom/mac.md

```diff
@@ -0,0 +1,29 @@
+# [Atom](../atom)
+
+## Duplicate selection: <kbd>⌘</kbd><kbd>d</kbd>
+
+There are two behaviors, and both come in handy.
+
+**First behavior:** Place your cursor within what you want to select, and press the shortcut. It will select the full alphanumeric sequence as your first selection. When you press it again to duplicate the selection, it defaults to selecting only **whole-word occurrences** of your current selection:
+
+![dup_selection_whole_word](http://i.imgur.com/0LqDNdt.png)
+
+**Second behavior:** By dragging to highlight, or double clicking, it will include each occurrence of that alphanumeric sequence, regardless of its context:
+
+![dup_selection_each_occurrence](http://i.imgur.com/evLAe1B.png)
+
+## Duplicate line: <kbd>⌘</kbd><kbd>Shift</kbd><kbd>d</kbd>
+
+Place your cursor anywhere, and it duplicates the current line. Your cursor remains in the lower line.
+
+## Open Project File: <kbd>⌘</kbd><kbd>Shift</kbd><kbd>t</kbd>, also <kbd>⌘</kbd><kbd>Shift</kbd><kbd>p</kbd>
+
+Press it, and you can start typing the name of the file immediately.
+
+## Move selected line(s) up and down: <kbd>⌘</kbd><kbd>Ctrl</kbd><kbd>↑</kbd>/<kbd>↓</kbd>
+
+## Needed
+- Split panes
```

Comment:
Windows: <kbd>ctrl</kbd><kbd>k</kbd> and then arrow in the direction to split (e.g. <kbd>→</kbd>)